ConsoleApp4/ConsoleApp4/Program.cs

```diff
@@ -0,0 +1,193 @@
+            }
+        }
+        public class QA
+        {
+
+            public string Question { get; set; }
+            public string Answer1 { get; set; }
+            public string Answer2 { get; set; }
+            public string Voter { get; set; }
+            public string Choice { get; set; }
+            public QA(string question="", string answer1="", string answer2="", string voter = "", string choice = "")
+            {
+                this.Question = question;
+                this.Answer1 = answer1;
+                this.Answer2 = answer2;
+                this.Voter = voter;
+                this.Choice = choice;
+            }
+            public void ShowQuestion()
+            {
+                if (Question == null)
+                {
+                    Console.WriteLine("No Question!");
+                    return;
+                }
+
+                Console.WriteLine("Question: {0}", Question);
+            }
+        }
+
+
+
+        static void Main()
+        {
+
+      QList ql = new QList();
+
+            ql.Add(new QA("0", "0", "0"));
+
+
+            while (true)
+            {
+                Menu( ref ql);
+            }
+        }
+        static void Menu( ref QList ql )
+        {
+
+
+
+            Console.Clear();
+            Console.WriteLine("\tMenu");
+            Console.WriteLine("\t1. Create poll");
+            Console.WriteLine("\t2. Show poll results");
+            Console.WriteLine("\t3. Vote for something");
+            Console.WriteLine("\t4. Show Result and Exit");
+
+            ConsoleKeyInfo ck = Console.ReadKey();
+
+            switch (ck.Key)
+            {
+                case ConsoleKey.D1:
+                case ConsoleKey.NumPad1:
+                    EnterPoolQuestion( ref ql);
+                    break;
+                case ConsoleKey.D2:
+                case ConsoleKey.NumPad2:
+                    ShowPollResult( ref ql);
+                    break;
+                case ConsoleKey.D3:
+                case ConsoleKey.NumPad3:
+                    VoteForSomething( ref ql);
+                    break;
+                case ConsoleKey.D4:
+                case ConsoleKey.NumPad4:
+                    ShowPollResultAndExit( ref ql);
+                    Exit();
+                    break;
+            }
+        }
+
+        static void EnterPoolQuestion(ref QList ql)
+        {
+            Console.Clear();
+            Console.WriteLine("Enter the question...\r\n");
+            Console.Write("Question: ");
+            string? question = Console.ReadLine();
+
+            Console.Write("Enter the option...\r\n0. Option ");
+            var answer1 = Console.ReadLine();
+
+            Console.Write("1. Answer ");
+            var answer2 = Console.ReadLine();
+
+            QList qlx=new QList();
+            
+                ql.Add(new QA( question, answer1, answer2 ));
+            
+
+            ql.ElementAt(0).Answer1= answer1;
```

Comment:
better to assign another variable value new QA(...) and then use it instead of ElementAt